bin/build.sh

```diff
@@ -2,17 +2,46 @@
 
 wca_url="https://worldcubeassociation.org/"
 
-# Find all Markdown files and build PDFs out of them.
+# Find formal and legal Markdown files and build PDFs out of them.
 find 'documents' -name '*.md' | while read file; do
+  echo "Processing $file, vanilla style"
+
   pdf_name="${file%.md}.pdf"
 
   sed -r "s#wca\{([^}]*)\}#$wca_url\1#g" "$file" | # Replace wca{...} with absolute WCA URL.
   pandoc | # Markdown -> HTML
   wkhtmltopdf --encoding 'utf-8' --user-style-sheet 'assets/style.css' -T 15mm -B 15mm -R 15mm -L 15mm --quiet - "$pdf_name" # HTML -> PDF
 done
 
-# Move pdf files to build directory
+compile_date=$(date '+%Y-%m-%d')
+
+# Find educational Markdown files and build PDFs out of them.
+find 'edudoc' -name '*.md' | while read file; do
+  echo "Processing $file, edudoc style"
```

Comment:
For log output, please see my other comment at L7: https://github.com/thewca/wca-documents/pull/120/files#r344507366